### PR TITLE
Add view and inference demo to SyMPC share tensor

### PR DESCRIPTION
**Pull request by gmuraru**
## Description
Add the view operation to SyMPC share tensor and also add an inference demo

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Running inference on 

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: examples/homomorphic-encryption/Tutorial_2_TenSEAL_Syft_Data_Owner.ipynb [hunk 1]
```diff
@@ -330,7 +330,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.5"
+   "version": "3.7.3"
```
Comment (LaRiffle) [inline examples/homomorphic-encryption/Tutorial_2_TenSEAL_Syft_Data_Owner.ipynb:333]: Don't commit those changes

File: examples/secure-multi-party-computation/Duet/1-DS-1-DO/POC-MPCTensor-Duet-Party2-DO.ipynb [hunk 2]
```diff
@@ -452,9 +452,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "venv",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "venv"
+   "name": "python3"
```
Comment (LaRiffle) [inline examples/secure-multi-party-computation/Duet/1-DS-1-DO/POC-MPCTensor-Duet-Party2-DO.ipynb:457]: same (you can you `git add -i` to cherry pick what you want to add to your commit)

**Review by LaRiffle — CHANGES_REQUESTED** [review]
Great job!

A few notes on the NB code:

Can you update the model structure as such? (I've moved the 2nd relu)
```python
class Net(sy.Module):
    def __init__(self, torch_ref):
        super(Net, self).__init__(torch_ref=torch_ref)
        self.conv1 = self.torch_ref.nn.Conv2d(
             in_channels=1, out_channels=3, kernel_size=5
        )
        self.fc1 = self.torch_ref.nn.Linear(1728, 128)
        self.fc2 = self.torch_ref.nn.Linear(128, 10)
 
    def forward(self, x):
        x = self.conv1(x)
        x = self.torch_ref.nn.functional.relu(x)
        x = x.view(1, -1)
        x = self.fc1(x)
        x = self.torch_ref.nn.functional.relu(x)
        x = self.fc2(x)
        return x
```

and `model.eval()` instead of `model.train(False)` ?

File: examples/secure-multi-party-computation/Duet/1-DS-1-DO/POC-MPCTensor-Duet-Party2-DO.ipynb (hunk 2 above)
Comment (gmuraru) [reply examples/secure-multi-party-computation/Duet/1-DS-1-DO/POC-MPCTensor-Duet-Party2-DO.ipynb:457]: > Great job!
> 
> A few notes on the NB code:
> 
> Can you update the model structure as such? (I've moved the 2nd relu)
> 
> ```python
> class Net(sy.Module):
>     def __init__(self, torch_ref):
>         super(Net, self).__init__(torch_ref=torch_ref)
>         self.conv1 = self.torch_ref.nn.Conv2d(
>              in_channels=1, out_channels=3, kernel_size=5
>         )
>         self.fc1 = self.torch_ref.nn.Linear(1728, 128)
>         self.fc2 = self.torch_ref.nn.Linear(128, 10)
>  
>     def forward(self, x):
>         x = self.conv1(x)
>         x = self.torch_ref.nn.functional.relu(x)
>         x = x.view(1, -1)
>         x = self.fc1(x)
>         x = self.torch_ref.nn.functional.relu(x)
>         x = self.fc2(x)
>         return x
> ```
> 
> and `model.eval()` instead of `model.train(False)` ?

Yep!

**Review by LaRiffle — APPROVED** [review]
🚀 🚀 🚀
